README.md

```diff
@@ -1,4 +1,5 @@
-![sotopia](figs/title.png)
+<p align="center"><img src="figs/title.png" width="300"></p>
```

Comment:
The icon looks a little bit weird from my perspective. Should we extend it to a rectangle one by filling more background in width?